model/clusters_mgmt/v1/gcp_flavour_type.model

```diff
@@ -25,5 +25,13 @@ struct GCPFlavour {
 	ComputeInstanceType String
 
 	// GCP default instance type for the infra volume.
-	InfraInstanceType String
+	
```

Comment:
Did you intend to remove the InfraInstanceType?

Reply:
no thanks!